app/Http/Controllers/QuestionController.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Category;
+use App\Question;
+use App\Choice;
+
+use Illuminate\Http\Request;
+
+class QuestionController extends Controller
+{
+    //
+
+    public function admin($id)
+    {
+        $category = Category::find($id);
+        // $questions = Question::where('category_id', $id)->get();
+        // $choices = Choice::where('question_id', $questions->id)->get();
+        $questions = $category->question()->get();
+        // $choices = $questions->choices()->get();
+        // foreach($questions as $question){
+        //     $choices = $question->choices()->get();
+        // }
+        // foreach(Category::find($id)->question() as $question){
+        //     $choices = $question->choices()->get();
+        // }
+        // $choices = Choice::where('question_id', $id)->get();
+        // $choices = Choice::find('question_id')->get();
+        // $choices = Choice::first();
+        // $choices = Question::all();
+        // $choices->choices()->get();
+        // $choices = Choice::with('question_id')->get();
+        // $choices = $questions->choices->first();
+        // $choices = DB::tabel('choices')->where('question_id', $id)->get();
+        // $choices = DB::tabel('questions')->where('id', $id)->get();
+        // $choices = $questions->choices()->where('question_id', $id)->first();
+        // $choices = Question::with('id')->get();
+        
+        
+        return view('only_admin.admin_questions', compact('category', 'questions', 'choices'));
```

Comment:
In this code, are you still going to use the commented functions? If not then you can erase them :)